### PR TITLE
added xsg.testnet.json

### PR DESCRIPTION
**Pull request by Gumbees**
I think testnet and mainnet us the same ports. I just added this for ease of use for new users.

### CODE REVIEW

File: pool_configs/examples/xsg.testnet.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "invalidAddress":"",
+    "_comment_invalidAddress": "Invalid addresses will be converted to the above",
+
+    "walletInterval": 2.5,
+    "_comment_walletInterval": "Used to cache coin stats",
+
+    "rewardRecipients": {
+        "": 1.0
```
Comment (egyptianbman) [inline pool_configs/examples/xsg.testnet.json:21]: 1%, you animal! :joy:

File: pool_configs/examples/xsg.testnet.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            "port": 16112,
+            "user": "rpcuser",
+            "password": "rpcpassword"
+        }
+    ],
+
+    "p2p": {
+        "enabled": false,
+        "host": "127.0.0.1",
+        "port": 16113,
```
Comment (egyptianbman) [inline pool_configs/examples/xsg.testnet.json]: This should be 26113

File: pool_configs/examples/xsg.testnet.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        "enabled": true,
+        "minConf": 10,
+        "paymentMode": "prop",
+        "_comment_paymentMode": "prop, pplnt",
+        "paymentInterval": 120,
+        "minimumPayment": 0.1,
+        "maxBlocksPerPayment": 3,
+        "daemon": {
+            "host": "",
+            "port": 16112 ,
```
Comment (egyptianbman) [inline pool_configs/examples/xsg.testnet.json]: This should be 26112

File: pool_configs/examples/xsg.testnet.json [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        }
+    },
+
+    "poolId": "main",
+    "_comment_poolId": "use it for region identification: eu, us, asia or keep default if you have one stratum instance for one coin",
+
+    "daemons": [
+        {
+            "host": "",
+            "port": 16112,
```
Comment (egyptianbman) [inline pool_configs/examples/xsg.testnet.json]: This should be 26112

**Review by egyptianbman — CHANGES_REQUESTED** [review]
The testnet ports aren't the same. Minor changes, thanks!

**Comment by Gumbees** [comment]
@egyptianbman corrected peer and rpc ports.

**Review by egyptianbman — APPROVED** [review]

**Pull request merged**